fe1-web/src/features/evoting/network/messages/__tests__/EndElection.test.ts

```diff
@@ -155,10 +156,21 @@ describe('EndElection', () => {
   });
 
   describe('computeRegisteredVotesHash', () => {
-    // It seems a bit silly to test this function apart from it not returning an error as
-    // we would simply write the same code here again?
-    const fn = () => EndElection.computeRegisteredVotesHash(mockElectionOpened);
-    expect(fn).not.toThrow();
-    expect(fn().valueOf()).toEqual('eYH10agf4Jvfs-rihA-9pG1j0lFPHnYeI9e9Vx-GQ6Q=');
+    it('return the correct result', () => {
+      // It seems a bit silly to test this function apart from it not returning an error as
+      // we would simply write the same code here again?
```

Comment:
I think this comment is left from a past version of the test (it now tests a value and does not only ensure that an error is not thrown), so you can delete it.